pkg/secretless/config/v2/config_env.go

```diff
@@ -0,0 +1,134 @@
+package v2
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/cyberark/secretless-broker/pkg/secretless/log"
+	"github.com/cyberark/secretless-broker/pkg/secretless/plugin"
+	"github.com/go-ozzo/ozzo-validation"
+)
+
+// ConfigEnv represents the runtime environment that will fulfill the services
+// requested by the Config.  It has a single public method, Prepare, that
+// ensures the runtime environment supports the requested services.
+type ConfigEnv struct {
+	logger       log.Logger
+	availPlugins plugin.AvailablePlugins
+	getFileInfo  func(name string) (os.FileInfo, error)
+	deleteFile   func(name string) error
```

Comment:
Field names should probably be sorted

Reply:
Type-aliasing might increase readability for these two func types if it's feasible